apps/librelingo_json_export/tests/test_challenges_clean_word.py

```diff
@@ -1,29 +1,27 @@
-from librelingo_utils import clean_word
-
-
-def test_empty_string():
-    assert clean_word("") == ""
-
-
-def test_removes_parentheses():
-    assert clean_word("(Hello there") == "Hello there"
-
-
-def test_removes_comma():
-    assert clean_word("This is,") == "This is"
-
-
-def test_doesnt_remove_parts_of_word_1():
-    assert clean_word("Mother-in-law") == "Mother-in-law"
-
-
-def test_doesnt_remove_parts_of_word_2():
-    assert clean_word("It's him") == "It's him"
-
-
-def test_removes_exclamation_mark():
-    assert clean_word("It's his mother-in-law!") == "It's his mother-in-law"
+import collections
+import pytest
 
+from librelingo_utils import clean_word
 
-def test_weird_english_posessive():
-    assert clean_word("cats'") == "cats'"
+TCData = collections.namedtuple("TCData", ["input_str", "expected", "name"])
```

Comment:
hmm it's weird for me to create a namedtuple just for this. though I definitely see the value in having fixed labels/names for each field instead of destructuring them "blindly" 

that being said it still feels weird. 

have you taken this idea from somewhere? :thinking:

Reply:
in any case, if we keep it maybe I'd prefer the name 

```suggestion
TestCaseData = collections.namedtuple("TestCaseData", ["input_str", "expected", "name"])
```

or 

```suggestion
TestConfig = collections.namedtuple("TestConfig", ["input_str", "expected", "name"])
```

or 


```suggestion
Example = collections.namedtuple("Example", ["input_str", "expected", "name"])
```

Reply:
> hmm it's weird for me to create a namedtuple just for this. though I definitely see the value in having fixed labels/names for each field instead of destructuring them "blindly"
> 
> that being said it still feels weird.
> 
> have you taken this idea from somewhere? thinking

Why _just for this_? I think we both agree that seeing a direct link between the current test data set and its _name_ is extremely valuable (some of the test cases in the original code had meaningful names). The naive way of using `@pytest.mark.parametrize` with specifying the `ids` is very tedious.

Defining a `TestCaseDataType` and having a _list_ of objects of it is appears to me as a standard way of parametrising tests in `GTest` in `C++`.

I think that using a `namedtuple` here is a good compromise between using a standard `tuple` and defining a proper class (which would be an overkill, since... there is `namedtuple`).

Reply:
nice, I really like it now!